policyengine_us/variables/gov/states/il/tax/income/credits/il_child_tax_credit.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class il_child_tax_credit(Variable):
```

Comment:
More aligned to other states right?
```suggestion
class il_ctc(Variable):
```